lib/msal-common/src/error/InteractionRequiredAuthError.ts

```diff
@@ -32,6 +34,8 @@ const InteractionRequiredAuthErrorMessages = {
         "The requested account is not available in the native broker. It may have been deleted or logged out. Please sign-in again using an interactive API.",
     [InteractionRequiredAuthErrorCodes.refreshTokenExpired]:
         "Refresh token has expired.",
+    [InteractionRequiredAuthErrorCodes.badToken]:
+        "Server returned invalid_grant because of an unusable refresh token was used. Refresh token must be removed from cache.",
```

Comment:
Cache clearing is performed by MSAL so this bit doesn't need to be suggested.

```suggestion
        "Identity provider returned bad_token due to an expired or invalid refresh token. Please invoke an interactive API to resolve.",
```